lib/reevoo_mark_utils.php

```diff
@@ -30,7 +30,10 @@ function getTrkref($options = array()) {
   }
 
   function getPaginationParams($options = array()) {
-    $page = self::presenceKey($_GET, 'reevoo_page', 1);
+    $page = isset($options['page'])
```

Comment:
Do you just want it to use 'reevoo_page' instead of 'page'? and also 'reevoo_' prefixed to the other options that I've made?